src/lib/Components/Bidding/Screens/ConfirmBid.tsx

```diff
@@ -322,21 +298,14 @@ export class ConfirmBid extends React.Component<ConfirmBidProps, ConfirmBidState
             <BidInfoRow label="Max bid" value={this.props.bid.display} onPress={() => this.goBackToSelectMaxBid()} />
 
             {requiresPaymentInformation ? (
-              <View>
-                <Divider mb={2} />
-                <BidInfoRow
-                  label="Credit Card"
-                  value={token && this.formatCard(token)}
-                  onPress={() => this.presentCreditCardForm()}
-                />
-                <Divider mb={2} />
-                <BidInfoRow
-                  label="Billing address"
-                  value={billingAddress && this.formatAddress(billingAddress)}
-                  onPress={() => this.presentBillingAddressForm()}
-                />
-                <Divider mb={2} />
-              </View>
+              <PaymentInfo
+                navigator={this.props.navigator}
+                onCreditCardAdded={this.onCreditCardAdded.bind(this)}
+                onBillingAddressAdded={this.onBillingAddressAdded.bind(this)}
+                billingAddress={this.state.billingAddress}
+                creditCardFormParams={this.state.creditCardFormParams}
+                creditCardToken={this.state.creditCardToken}
+              />
```

Comment:
I'd rather make a `<CreditCardRow>` and `<BillingAddressRow>` components. Monolith components are a typical source of complexity and having a sinlge component to reduce 2 occurances in `<ConfirmBid>` and `<Registration>` seems less compeling:

```ts
<View>
  <CreditCardRow
    navigator={this.props.navigator}
    onCreditCardAdded={this.onCreditCardAdded.bind(this)}>

  <Divider mb={2} />

  <BillingAddressRow
    navigator={this.props.navigator}
    onCreditCardAdded={this.onCreditCardAdded.bind(this)}>

  <Divider mb={2} />
</View>
```

Reply:
Yeah, the main reason I thought this was okay was that we don't have any cases where you would have these components in isolation (i.e. you always have to have a billing address with a credit card, and vice versa). 

However, I don't feel strongly. Happy to extract.